bin/cli.js

```diff
@@ -95,18 +88,15 @@ const writeToDisk = function (fileName, data) {
 const pwMetrics = new PWMetrics(options.url, options);
 pwMetrics.start()
   .then(data => {
-    if (options.flags.output) {
-      //serialize accordingly
-      switch (options.flags.output) {
-        case 'json':
-          data = JSON.stringify(data, null, 2) + '\n';
-          break;
-        default:
-          break;
-      }
-      //output to file.
-      if (options.flags.outputPath.length) return writeToDisk(options.flags.outputPath, data);
-      else data && process.stdout.write(data);
+    if (options.flags.json) {
+      // serialize accordingly
+      data = JSON.stringify(data, null, 2) + '\n';
+      // output to file.
+      if (options.flags.outputPath != 'stdout')
+        return writeToDisk(options.flags.outputPath, data);
+      // output to stdout
+      else if (data)
+        process.stdout.write(data);
```

Comment:
The ```process.exit(0)``` is in the ```then/catch```callbacks. This function is merely for data output processing (serialize and write to stream).